### PR TITLE
Feature/tao 7191/create sync log service

### PR DESCRIPTION
**Pull request by SashaTAO**
Create SyncLogService
Depends on https://github.com/oat-sa/extension-tao-sync/pull/85/files

### CODE REVIEW

File: model/SyncLog/SyncLogServiceInterface.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+     * @param $boxId
+     * @return SyncLogEntity
+     */
+    public function getBySyncIdAndBoxId($syncId, $boxId);
+
+    /**
+     * Get total amount of synchronization logs by provided filters.
+     *
+     * @param SyncLogFilter $filter
+     * @return array
```
Comment (SergiiTao) [inline model/SyncLog/SyncLogServiceInterface.php]: it should be an integer, i guess

File: model/SyncLog/SyncLogService.php [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        }
+
+        return $this->storage;
+    }
+
+    /**
+     * Create new record in storage implementation.
+     *
+     * @param SyncLogEntity $entity
+     * @return mixed|void
```
[the remark this answers is not in the document]
Comment (SashaTAO) [reply model/SyncLog/SyncLogService.php]: Fixed to return correct type returned by storage

File: model/SyncLog/SyncLogService.php [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+            throw new SyncLogEntityNotFound($e->getMessage());
+        }
+
+    }
+
+    /**
+     * Get total amount of synchronization logs by provided filters.
+     *
+     * @param SyncLogFilter $filter
+     * @return array
```
[the remark this answers is not in the document]
Comment (SashaTAO) [reply model/SyncLog/SyncLogService.php]: Fixed

File: config/default/SyncLogService.conf.php [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+
+use oat\taoSync\model\SyncLog\SyncLogService;
+use oat\taoSync\model\SyncLog\Storage\RdsSyncLogStorage;
+
+return $syncLogService = new SyncLogService([SyncLogService::OPTION_STORAGE => RdsSyncLogStorage::SERVICE_ID]);
```
[the remark this answers is not in the document]
Comment (SashaTAO) [reply config/default/SyncLogService.conf.php]: It was copy/paste from update script. Fixed.

**Pull request merged**